config/initializers/slack.rb

```diff
@@ -1,3 +1,3 @@
 Slack.configure do |config|
-  config.token = 'xoxb-27846047942-A69pQBzqSHv9aIrwMUFjGSWP'
+  config.token = 'xoxb-27846047942-vBiLMQhunVZhlI7PS12jGrPH'
```

Comment:
Can we leverage on `figaro` and `environmental variables` for this? I don't like playing with bad people. 😄